static/js/settings_linkifiers.js

```diff
@@ -33,6 +36,24 @@ function sort_url(a, b) {
     return compare_by_index(a, b, 1);
 }
 
+function open_linkifier_edit_form(filter_id) {
+    const realm_filters_list = page_params.realm_filters;
+    const realm_filter = realm_filters_list.find((realm_filter) => realm_filter[2] === filter_id);
+    const html = render_admin_linkifier_edit_form({
+        filter_id,
+        pattern: realm_filter[0],
+        url_format_string: realm_filter[1],
+    });
+    const div = $(html);
+    div.find("input").css("width", "400px");
+    div.find(".alert").css("margin-top", "10px");
```

Comment:
These should be done via CSS rules -- wherever possible we avoid doing inline styles like this from JavaScript.